basis-library/arrays-and-vectors/slice0.sig

```diff
@@ -10,8 +10,8 @@
 
 signature PRIM_SLICE = 
    sig
-      type 'a elt
       type 'a sequence
+      type 'a elt
```

Comment:
All of these `basis-library/arrays-and-vectors/*` changes seem unrelated to the type changes, so I've reverted them.